### PR TITLE
fix: don't render list items as task items

### PR DESCRIPTION
**Pull request by tavin**

### CODE REVIEW

File: src/components/listItem.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export function ListItem({
   line?: number;
   children?: any[];
 }): JSX.Element {
-  if (checked === null) {
+  if (checked === undefined) {
```
Comment (rowanc1) [inline src/components/listItem.tsx:48]: `if (checked == null) {` would catch both cases!

Comment (agoose77) [inline src/components/listItem.tsx:48]:
```suggestion
  if (checked == null) {
```

Comment (agoose77) [reply src/components/listItem.tsx:48]: This is a silly bug! Nice find.

Comment (tavin) [reply src/components/listItem.tsx]: it's academic, but under what circumstances can `checked` actually be null?

**Pull request merged**